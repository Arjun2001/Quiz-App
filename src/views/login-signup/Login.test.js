import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import Login from './Login_signup';

describe('Blog component', () => {
    it('div rendering', () => {
        const wrapper = shallow(<Login />);
        wrapper.find('div.container-1');
    });
    it('button rendering', () => {
        const wrapper = shallow(<Login />);
        wrapper.find('button.switch-button');
    });
    it('matches the snapshot', () => {
        const tree = renderer.create(<Login />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});