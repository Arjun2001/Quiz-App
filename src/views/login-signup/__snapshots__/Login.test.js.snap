// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog component matches the snapshot 1`] = `
<div>
  <div
    className="container1"
  >
    <div
      className="overlay"
      id="overlay"
    >
      <div
        className="sign-in"
        id="sign-in"
      >
        <h1>
          Welcome Back!
        </h1>
        <p>
          To keep connected with us please login with your personal info
        </p>
        <button
          className="switch-button"
          id="slide-right-button"
        >
          Sign In
        </button>
      </div>
      <div
        className="sign-up"
        id="sign-up"
      >
        <h1>
          Hello, Friend!
        </h1>
        <p>
          Enter your personal details and start a journey with us
        </p>
        <button
          className="switch-button"
          id="slide-left-button"
        >
          Sign Up
        </button>
      </div>
    </div>
    <div
      className="form"
    >
      <div
        className="sign-in"
        id="sign-in-info"
      >
        <h1>
          Sign In
        </h1>
        <div
          className="social-media-buttons"
        >
          <button
            className="icon"
            onClick={[Function]}
          >
            <svg
              viewBox="0 0 24 24"
            >
              <path
                d="M17,2V2H17V6H15C14.31,6 14,6.81 14,7.5V10H14L17,10V14H14V22H10V14H7V10H10V6A4,4 0 0,1 14,2H17Z"
                fill="#000000"
              />
            </svg>
          </button>
          <button
            className="icon"
            onClick={[Function]}
          >
            <svg
              viewBox="0 0 24 24"
            >
              <path
                d="M23,11H21V9H19V11H17V13H19V15H21V13H23M8,11V13.4H12C11.8,14.4 10.8,16.4 8,16.4C5.6,16.4 3.7,14.4 3.7,12C3.7,9.6 5.6,7.6 8,7.6C9.4,7.6 10.3,8.2 10.8,8.7L12.7,6.9C11.5,5.7 9.9,5 8,5C4.1,5 1,8.1 1,12C1,15.9 4.1,19 8,19C12,19 14.7,16.2 14.7,12.2C14.7,11.7 14.7,11.4 14.6,11H8Z"
                fill="#000000"
              />
            </svg>
          </button>
          <button
            className="icon"
            onClick={[Function]}
          >
            <svg
              className="bi bi-telephone"
              fill="currentColor"
              height="16"
              viewBox="0 0 16 16"
              width="16"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M3.654 1.328a.678.678 0 0 0-1.015-.063L1.605 2.3c-.483.484-.661 1.169-.45 1.77a17.568 17.568 0 0 0 4.168 6.608 17.569 17.569 0 0 0 6.608 4.168c.601.211 1.286.033 1.77-.45l1.034-1.034a.678.678 0 0 0-.063-1.015l-2.307-1.794a.678.678 0 0 0-.58-.122l-2.19.547a1.745 1.745 0 0 1-1.657-.459L5.482 8.062a1.745 1.745 0 0 1-.46-1.657l.548-2.19a.678.678 0 0 0-.122-.58L3.654 1.328zM1.884.511a1.745 1.745 0 0 1 2.612.163L6.29 2.98c.329.423.445.974.315 1.494l-.547 2.19a.678.678 0 0 0 .178.643l2.457 2.457a.678.678 0 0 0 .644.178l2.189-.547a1.745 1.745 0 0 1 1.494.315l2.306 1.794c.829.645.905 1.87.163 2.611l-1.034 1.034c-.74.74-1.846 1.065-2.877.702a18.634 18.634 0 0 1-7.01-4.42 18.634 18.634 0 0 1-4.42-7.009c-.362-1.03-.037-2.137.703-2.877L1.885.511z"
              />
            </svg>
          </button>
        </div>
        <p
          className="small"
        >
          or use your email account:
        </p>
        <form
          id="sign-in-form"
          onSubmit={[Function]}
        >
          <input
            name="roll_no"
            placeholder="Roll Number"
            required={true}
            type="text"
          />
          <input
            name="password"
            placeholder="Password"
            required={true}
            type="password"
          />
          <div
            className="checkbox"
          >
            <input
              id="rememberMe"
              name="rememberMe"
              type="checkbox"
            />
             Remember me
          </div>
          <p
            className="forgot-password"
            onClick={[Function]}
            style={
              Object {
                "pointer": "cursor",
              }
            }
          >
            Forgot your password?
          </p>
          <button
            className="control-button in"
            type="submit"
          >
            Sign In
          </button>
        </form>
      </div>
      <div
        className="sign-up"
        id="sign-up-info"
      >
        <h1>
          Create Account
        </h1>
        <form
          id="sign-up-form"
          onSubmit={[Function]}
        >
          <input
            name="roll_no"
            placeholder="Roll Number"
            required={true}
            size="6"
            type="text"
          />
          <input
            max="2021-04-13"
            name="dob"
            required={true}
            type="date"
          />
          <input
            name="phone_no"
            pattern="[0-9]{10}"
            placeholder="Phone Number"
            required={true}
            type="number"
          />
          <input
            name="email"
            placeholder="Email"
            required={true}
            type="email"
          />
          <input
            name="password"
            placeholder="Password"
            required={true}
            type="password"
          />
          <button
            className="control-button up"
            type="submit"
          >
            Sign Up
          </button>
        </form>
      </div>
    </div>
  </div>
</div>
`;
